### PR TITLE
Generating SDK with the latest Typespec commit ID

### PR DESCRIPTION
**Pull request by hamshavathimunibyraiah**
Generating SDK with the latest Typespec commit ID that has the finaloperation of the LRO call to return "TranslationStatus": https://github.com/Azure/azure-rest-api-specs/pull/30970

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
you can delete the samples in generated folder.

File: sdk/translation/azure-ai-translation-document/tsp-location.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 directory: specification/translation/Azure.AI.DocumentTranslation
-commit: 27a9398801386caaba2df7e1a4d1a8abd19e3789
-repo: Azure/azure-rest-api-specs
+commit: a2b9e584b3c6aafa1d3a977b21b0d024f28cfe1c
+repo: hamshavathimunibyraiah/azure-rest-api-specs
```
Comment (catalinaperalta) [inline sdk/translation/azure-ai-translation-document/tsp-location.yaml:3]: Isnt this commit from the Azure/azure-rest-api-specs repo?

Comment (weidongxu-microsoft) [reply sdk/translation/azure-ai-translation-document/tsp-location.yaml:3]: I think https://github.com/Azure/azure-rest-api-specs/pull/30970 works. Maybe your configure here is not correct.

Mine test https://github.com/Azure/azure-sdk-for-java/pull/42338/files

Comment (hamshavathimunibyraiah) [reply sdk/translation/azure-ai-translation-document/tsp-location.yaml]: didn't think it mattered, but updated

**Review by samvaity — APPROVED** [review]

**Review by srnagar — APPROVED** [review]

**Pull request merged**